### PR TITLE
Added retrieval of precision and scale information for decimal types...

### PR DESCRIPTION
**Pull request by magneto-81**
Added retrieval of precision and scale information for decimal types to the datareader schema.

### CODE REVIEW

File: DuckDB.NET.Data/DuckDBDataReader.cs [hunk 1]
```diff
@@ -307,6 +309,18 @@ public override DataTable GetSchemaTable()
             rowData[2] = GetFieldType(i);
             rowData[3] = -1;
             rowData[4] = true;
+
+            if (vectorReaders[i] is DecimalVectorDataReader dvdr)
```
Comment (Giorgi) [inline DuckDB.NET.Data/DuckDBDataReader.cs:313]: Can you give it a more meaningful name?

File: DuckDB.NET.Data/Internal/Reader/DecimalVectorDataReader.cs [hunk 2]
```diff
@@ -61,4 +63,14 @@ private decimal GetDecimal(ulong offset)
             default: throw new DuckDBException($"Invalid type {DuckDBType} ({(int)DuckDBType}) for column {ColumnName}");
         }
     }
+
+    internal byte GetScale()
```
Comment (Giorgi) [inline DuckDB.NET.Data/Internal/Reader/DecimalVectorDataReader.cs:67]: I think it's better to turn these fields into readonly properties.

File: DuckDB.NET.Data/DuckDBDataReader.cs (hunk 1 above)
Comment (magneto-81) [reply DuckDB.NET.Data/DuckDBDataReader.cs]: Change it.

File: DuckDB.NET.Data/Internal/Reader/DecimalVectorDataReader.cs (hunk 2 above)
Comment (magneto-81) [reply DuckDB.NET.Data/Internal/Reader/DecimalVectorDataReader.cs]: Change it.

**Pull request merged**